example/android/app/build.gradle

```diff
@@ -27,16 +26,22 @@ android {
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.google.code.findbugs:jsr305:3.0.0'
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation 'com.google.code.findbugs:jsr305:3.0.0'
 }
 
 task ndkBuild(type: Exec) {
-    File ndkDir = project.android.ndkDirectory
-    def ndkBuildExecutable = new File(ndkDir, 'ndk-build')
+    def ndkDir = project.android.ndkDirectory
+    if (ndkDir) {
+        def ndkBuildExecutable = new File(ndkDir, 'ndk-build')
```

Comment:
This "ndkBuildExecutable" here is also used by "line 50" below, and wrapping it in the `if` statement makes it invisible outside and cause error when executing "lien 50": `if (!ndkBuildExecutable.exists()) {`